test/api/students_api_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+require 'user'
+
+class StudentsApiTest < ActiveSupport::TestCase
+  include Rack::Test::Methods
+  include TestHelpers::AuthHelper
+  include TestHelpers::JsonHelper
+
+  def app
+    Rails.application
+  end
+
+  def test_get_students_with_authentication
+    # Build admin user
+    adminUser = FactoryGirl.build(:user, :admin)
+
+    # The get that we will be testing.
+    get with_auth_token '/api/students/?unit_id=1',adminUser
```

Comment:
You cannot rely upon units existing either... these need to be built.

Reply:
I am sorry. I have fixed it.